backend/tests/v5/unit/processors/teamspaces/projects/models/containers.test.js

```diff
@@ -205,9 +220,34 @@ const testGetContainerStats = () => {
 	});
 };
 
+const testGetRevisions = () => {
+	describe('Get container revisions', () => {
+		test('should return non-void revisions if the container exists', async () => {
+			const res = await Containers.getRevisions('teamspace', 1, false);
+			expect(res).toEqual(model1Revisions.filter((r) => !r.void));
+		});
+		test('should return all revisions if the container exists', async () => {
+			const res = await Containers.getRevisions('teamspace', 1, true);
+			expect(res).toEqual(model1Revisions);
+		});
+	});
+};
+
+const testUpdateRevisionStatus = () => {
+	const revision3 = model1Revisions.find((r) => r._id === 3);
+	describe('Get container revisions', () => {
+		test('should update revision if the request body is boolean', async () => {
+			await Containers.updateRevisionStatus('teamspace', 1, revision3._id, false);
+			expect(revision3.void).toEqual(false);
```

Comment:
again, you're checking your mock implementation 😆 

Just spy on the update call and check it's being called + with right parameters.